language/diem-tools/diem-keygen/src/main.rs

```diff
@@ -0,0 +1,15 @@
+use diem_crypto::ValidCryptoMaterialStringExt;
+use diem_keygen::KeyGen;
+
+fn main() {
+    let mut keygen = KeyGen::from_os_rng();
+    let (privkey, pubkey) = keygen.generate_keypair();
+
+    println!("Private Key:");
```

Comment:
?

Reply:
Well, since the test harness does not currently generate keys for you, you'll have to do it manually a lot when writing tests. I made this executable to make that work slightly less tedious.